src/lib/common/Format.h

```diff
@@ -31,6 +31,15 @@
 
 /* ****************************************************************************
 *
```

Comment:
I don't believe there is anything additional for CHANGES_NEXT_RELEASE.
The bug fixes in rendering aren't really nugfixes as it was dead code until this very PR and the problems are never seen until this PR and fixed before being visible.
Have you seen anything else that might need mentioning in CHANGES_NEXT_RELEASE?